Services/CO.CDP.Person.WebApi/Api/Person.cs

```diff
@@ -106,6 +107,26 @@ await useCase.Execute(command)
                 operation.Responses["204"].Description = "Person deleted.";
                 return operation;
             });
+
+        app.MapPost("/persons/{personId}/associate",
+        async (Guid personId, AssociatePerson command, IUseCase<AssociatePerson, Model.Person> useCase) =>
+                 await useCase.Execute(command)
+                     .AndThen(organisation =>
+                         organisation != null
```

Comment:
According to the `IUseCase<AssociatePerson, Model.Person>` type, person will never be `null`.